Assets/Scripts/AI/ThirdPersonCharacter.cs

```diff
@@ -0,0 +1,225 @@
+using UnityEngine;
+
+namespace UnityStandardAssets.Characters.ThirdPerson
+{
+	[RequireComponent(typeof(Rigidbody))]
+	[RequireComponent(typeof(CapsuleCollider))]
+	[RequireComponent(typeof(Animator))]
+	public class ThirdPersonCharacter : MonoBehaviour
+	{
+		[SerializeField] float m_MovingTurnSpeed = 360;
+		[SerializeField] float m_StationaryTurnSpeed = 180;
+		[SerializeField] float m_JumpPower = 12f;
+		[Range(1f, 4f)][SerializeField] float m_GravityMultiplier = 2f;
+		[SerializeField] float m_RunCycleLegOffset = 0.2f; //specific to the character in sample assets, will need to be modified to work with others
+		[SerializeField] float m_MoveSpeedMultiplier = 1f;
+		[SerializeField] float m_AnimSpeedMultiplier = 1f;
+		[SerializeField] float m_GroundCheckDistance = 0.1f;
+
+		Rigidbody m_Rigidbody;
+		Animator m_Animator;
+		bool m_IsGrounded;
+		float m_OrigGroundCheckDistance;
+		const float k_Half = 0.5f;
+		float m_TurnAmount;
+		float m_ForwardAmount;
+		Vector3 m_GroundNormal;
+		float m_CapsuleHeight;
+		Vector3 m_CapsuleCenter;
+		CapsuleCollider m_Capsule;
+		bool m_Crouching;
+
+
+		void Start()
+		{
+			m_Animator = GetComponent<Animator>();
+			m_Rigidbody = GetComponent<Rigidbody>();
+			m_Capsule = GetComponent<CapsuleCollider>();
+			m_CapsuleHeight = m_Capsule.height;
+			m_CapsuleCenter = m_Capsule.center;
+
+			m_Rigidbody.constraints = RigidbodyConstraints.FreezeRotationX | RigidbodyConstraints.FreezeRotationY | RigidbodyConstraints.FreezeRotationZ;
+			m_OrigGroundCheckDistance = m_GroundCheckDistance;
+		}
+
+
+		public void Move(Vector3 move, bool crouch, bool jump)
+		{
+
+			// convert the world relative moveInput vector into a local-relative
+			// turn amount and forward amount required to head in the desired
+			// direction.
+			if (move.magnitude > 1f) move.Normalize();
+			move = transform.InverseTransformDirection(move);
+			CheckGroundStatus();
+			move = Vector3.ProjectOnPlane(move, m_GroundNormal);
+			m_TurnAmount = Mathf.Atan2(move.x, move.z);
+			m_ForwardAmount = move.z;
+
+			ApplyExtraTurnRotation();
+
+			// control and velocity handling is different when grounded and airborne:
+			if (m_IsGrounded)
+			{
```

Comment:
You made everything one line in my comment above, but the if statements following this one are all braced. Gotta stay consistent here haha. If the block is only one line, we don't use the braces, but they stay on a new line

Reply:
Ok i'm guessing you picked this file off somewhere right? Because the style in this file was different from the others. Make sure to look through before you add it.